CONTRIBUTING.md

```diff
@@ -0,0 +1,74 @@
+# Contributor Guidelines
+
+Thank you for considering contributing to our project! 
+We appreciate your interest and value the contributions from our community.
+To ensure a smooth collaboration, we have established the following guidelines to help you understand the contribution process and make your experience as positive and effective as possible.
+
+## Advice for new contributors
+
+Familiarize yourself with the project's purpose, goals, and architecture.
+Explore the existing documentation, including the README file.
+Check the issue tracker and pull requests to see if your desired contribution is already being discussed or worked on.
+Join our community discussion channels to engage with other contributors and seek guidance if needed.
+
+## Code of Conduct
+
+Please adhere to our project's Code of Conduct when participating in any project-related activities, both online and offline.
+Treat all community members with respect and create a welcoming and inclusive environment.
+
+## Reporting Issues
+
+If you encounter a bug or have a suggestion, please submit an issue in the project's issue tracker.
```

Comment:
link?

Reply:
I haven't added any project specific links here to keep it common for all the repositories. The link for issue tracker and PRs is available in ReadME